### PR TITLE
chore(cocoapods): update version

### PR DESCRIPTION
**Pull request by petergrau**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [x] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

Will this also update the following image `https://github.com/bitrise-io/bitrise.io/blob/master/system_reports/osx-xcode-11.5.x.log` or does that require another PR?

### CODE REVIEW

**Comment by lmesz-bitrise** [comment]
Hello,

Thank you. for your contribution, it's cool :)
Please update the test that belongs to that role: https://github.com/bitrise-io/osx-box-bootstrap/blob/master/roles/ruby/tests/test_ruby.py#L14

Thank you

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## UPCOMING
 
+## `v2020_10_27_1`
+* `install cocoapds to the latest`
```
Comment (MrBlaise) [inline CHANGELOG.md:4]:
```suggestion
* `install cocoapods to the latest`
```

**Review by fehersanyi-bitrise — APPROVED** [review]
lgtm

**Review by MrBlaise — APPROVED** [review]

**Comment by MrBlaise** [comment]
@petergrau This won't affect existing stacks, just new stacks in the future. So to answer your question it will not update 11.5.

**Comment by petergrau** [comment]
> @petergrau This won't affect existing stacks, just new stacks in the future. So to answer your question it will not update 11.5.
> 
> EDIT: The reason is that this role is not part of the weekly update playbook, due to the way we provision the images.

@MrBlaise thanks for the answer! is there any way to get the new cocoapods version into an image with Xcode 11.5 ?

**Comment by MrBlaise** [comment]
> > @petergrau This won't affect existing stacks, just new stacks in the future. So to answer your question it will not update 11.5.
> > EDIT: The reason is that this role is not part of the weekly update playbook, due to the way we provision the images.
> 
> @MrBlaise thanks for the answer! is there any way to get the new cocoapods version into an image with Xcode 11.5 ?

@petergrau We have created a ticket to investigate the effect of upgrading existing stacks, this ticket will probably be scheduled in the next sprint and I will have more info then. For now I can't say if we can update 11.5, I'll keep you updated when we have more info.

**Pull request merged**